build/docs/classes/DocsBuilder.php

```diff
@@ -912,7 +918,6 @@ private function renderShape(
         ksort($members);
         foreach ($members as $name => $member) {
             $html->open('dt', 'param-def');
-            $html->elem('a', ['href' => '#' . $this->memberSlug($name)], '');
```

Comment:
To confirm, this was deleting the fragment that didn't exist and just sent you to the top of the page, right?  If so, just resolve this comment